en/asset/label-atlas.md

```diff
@@ -0,0 +1,32 @@
+# LabelAtlas Asset
+
+**LabelAtlas** asset is a user defined asset, it's used for configuring LabelAtlas.
+
+## Create LabelAtlas Asset
+
+In **Assets Panel**  right-click on a folder, and click the following context menu item **Create -> Label Atlas**. It will create a **LabelAtlas.labelatlas** asset.
```

Comment:
In **Assets Panel**  right-click on a folder, and click the following context menu item **Create -> Label Atlas**. It will create a **LabelAtlas.labelatlas** asset.
-->
In the **Assets Panel** right-click on a folder, and click the context menu item **Create -> Label Atlas**. It will create a **LabelAtlas.labelatlas** asset.